build/musicbrainz/Dockerfile

```
@@ -12,20 +12,26 @@ RUN curl -sSL https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
     apt-get update && \
     apt-get install -y \
         cpanminus \
-        build-essential \
+        bzip2 \
         gettext \
-        git-core \
+        g++ \
+        git \
+        # Needed for Cache in DB_File
         libdb-dev \
         libexpat1-dev \
         libicu-dev \
         liblocal-lib-perl \
         libpq-dev \
-        libxml2-dev \
         libssl-dev \
+        # Needed for XML::LibXML and lxml which is installed with mb-rngpy
```

`mb-rngpy` (thus `lxml`) is a dependency of `sir` (indexer), not musicbrainz server.

```suggestion
        # Needed for XML::LibXML
```